ksqldb-engine/src/test/java/io/confluent/ksql/integration/EndToEndIntegrationTest.java

```diff
@@ -123,6 +123,10 @@
       .withAdditionalConfig(
           KsqlConfig.SCHEMA_REGISTRY_URL_PROPERTY,
           "http://foo:8080")
+      .withAdditionalConfig(
+          KsqlConfig.KSQL_KEY_FORMAT_ENABLED,
```

Comment:
We're planning on enabling Avro keys soon, right? We should remember to pull these out when we do.

Reply:
Added a task to the project.